app/controllers/users_controller.rb

```diff
@@ -0,0 +1,21 @@
+class UsersController < ApplicationController
+  def welcome
+    render :welcome
+  end
```

Comment:
It is interesting that you chose to put the welcome page under the Users Controller. Is there a reason for doing this?